user/advanced-configuration/rpc-policy.md

```diff
@@ -17,6 +17,7 @@ Here's an example of an RPC policy file in dom0:
 ```
 [user@dom0 user ~]$ cat /etc/qubes-rpc/policy/qubes.FileCopy
 (...)
+@tag:work   @default    ask
```

Comment:
Wouldn't this supersede the rows below, since evaluation stops at the first match? Maybe I'm not understanding `@default`.

Reply:
Never mind. I just re-read https://github.com/QubesOS/qubes-issues/issues/4403, and I think I understand what the goal is now. (Still not sure if it's correct or will work, but I'll leave that up to Marek.)